source/empty/index.rst

```diff
@@ -116,13 +125,17 @@ Once you get your updated ``PO`` files, you'll have to compile them to ``MO`` fi
 Release script
 ^^^^^^^^^^^^^^
 
-A release script is provided in ``tools/release``. This is a "simple" `Python <http://python.org>`_ script; you should just have Python installed on your system (this is instaleld by defautl on most linux distributions).
+A release script is provided in ``vendor/bin/plugin-release``. This is a "simple" `Python <http://python.org>`_ script; you should just have Python installed on your system (this is installed by default on most linux distributions).
 
-Using just the defaults, the script will try to retrieve the latest tag in your git repository, and will propose you to release it:
+.. warning::
+
+   Note that for the moment, the release script is only compatible if you use semantic versionning!
+
+Using just the defaults, the script will try to retrieve the latest tag in your git repository, add third party dependencies and create a `Release` on the github project:
```

Comment:
Yes, in the github release section:
http://glpi-plugins.readthedocs.io/en/feature-external-tools/empty/index.html#github-release